packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -126,6 +133,48 @@ export function compileBlueprint(
 				: blueprint.login),
 		});
 	}
+
+	/**
+	 * Download WP-CLI. {{{
+	 * Hardcoding this in the compilt() function is a temporary solution
+	 * to provide the wpCLI step with the wp-cli.phar file it needs. Eventually,
+	 * each Blueprint step may be able to specify any pre-requisite resources.
+	 * Also, wp-cli should only be downloaded if it's not already present.
+	 */
+	const wpCliStepIndex = blueprint.steps?.findIndex(
+		(step) => typeof step === 'object' && step?.step === 'wp-cli'
+	);
+	if (wpCliStepIndex !== undefined && wpCliStepIndex > -1) {
+		if (!blueprint.phpExtensionBundles) {
+			blueprint.phpExtensionBundles = [];
+		}
+		if (!blueprint.phpExtensionBundles.includes('kitchen-sink')) {
+			blueprint.phpExtensionBundles.push('kitchen-sink');
+			console.warn(
+				`The WP-CLI step used in your Blueprint requires the iconv and mbstring PHP extensions. ` +
+					`However, you did not specify the kitchen-sink extension bundle. Playground will override your ` +
+					`choice and load the kitchen-sink PHP extensions bundle to prevent the WP-CLI step from failing. `
+			);
+		}
+		blueprint.steps?.splice(wpCliStepIndex, 0, {
```

Comment:
Let's make this request optional or else running this Blueprint step in `wp-now` will always trigger a fetch request.

Reply:
The step works on the web. Let's figure out node.js in a follow-up PR.